server/entity/MediaRequest.ts

```diff
@@ -1137,7 +1141,9 @@ export class MediaRequest {
           request: this,
           notifyAdmin,
           notifySystem,
-          notifyUser: notifyAdmin ? undefined : this.requestedBy,
+          notifyUser: Notification.MEDIA_AUTO_REQUESTED
+            ? this.requestedBy
+            : undefined,
```

Comment:
This will impact the notification logic as well. When this is set, the users are sent a notification.

Reply:
I pushed a new change that should correct this. Let me know what you think.